impl/client/src/main/javascript/web/util/_a11y.js

```diff
@@ -0,0 +1,92 @@
+/*!
+ * This program is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License, version 2.1 as published by the Free Software
+ * Foundation.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along with this
+ * program; if not, you can obtain a copy at http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html
+ * or from the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ *
+ * Copyright (c) 2002-2023 Hitachi Vantara..  All rights reserved.
+ */
+
+define(function () {
+
+  var keyCodes = {
+    backSpace: 8,
+    tab: 9,
+    enter: 13,
+    shift: 16,
+    escape: 27,
+    space: 32,
+    pageUp: 33,
+    pageDown: 34,
+    end: 35,
+    home: 36,
+    arrowLeft: 37,
+    arrowUp: 38,
+    arrowRight: 39,
+    arrowDown: 40,
+    delete: 46
+  };
+
+  var _isToggleButton = false;
```

Comment:
This module variable is not the best way to solve the issue. There are simpler, safer, cleaner solutions.

For example, create an internal function which receives a parameter that indicates the type of button:
```js
function makeAccessibleButton(elem, isToggleButton) {
   // ...
}

return {
    keyCodes: keyCodes,

    makeAccessibleActionButton: function(elem) { 
       makeAccessibleButton(elem, false);
    },

    makeAccessibleToggleButton: function(elem) { 
       makeAccessibleButton(elem, true);
    }
  };
```